mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js

```diff
@@ -636,7 +636,8 @@ GEOR.managelayers = (function() {
                 layerRecord.hasEquivalentWCS() : false,
             isVector = layer instanceof OpenLayers.Layer.Vector,
             isBaseLayer = layerRecord.get("opaque") || 
-                layer.transitionEffect === "resize";
+                layer.transitionEffect === "resize",
+            isSingleTile = layer.options.singleTile;
```

Comment:
singleTile is a layer property, see  https://github.com/georchestra/openlayers/blob/master/lib/OpenLayers/Layer/Grid.js#L68-L75